composer.json

```diff
@@ -43,16 +42,23 @@
         ]
     },
     "scripts": {
-        "ci": [
-            "@fix",
-            "@analyse",
-            "@test"
+        "qa": [
+            "linter src tests",
+            "codesniffer src tests"
         ],
-        "fix": "\"vendor/bin/php-cs-fixer\" fix",
-        "analyse": "\"vendor/bin/phpstan\" analyse --no-progress --level=max --configuration=phpstan.neon src tests",
-        "test": [
-            "\"vendor/bin/codecept\" build",
-            "\"vendor/bin/codecept\" run --debug"
+        "tester": [
+            "tester -s -p php --colors 1 -C tests/cases"
```

Comment:
This won't work. Unless you want to change the tests to nette/tester which doesn't really make much sense in this case.

Reply:
Yep, thanks.